src/components/Category/Category.js

```diff
@@ -1,14 +1,30 @@
-import React from "react";
+import React, { Component } from "react";
+import PropTypes from 'prop-types';
+
+import "./../Button/button.css";
 import "./category.css";
 
-const Category = ({ name }) => (
-  <div>
-    <label className="container">
-      {name}
-      <input type="checkbox" />
-      <span className="checkmark" />
-    </label>
-  </div>
-);
+class Category extends Component {
+  render(){
+    const { name, handleClick } = this.props;
+    return(
+      <div>
```

Comment:
Btw, just a small note - it nothing wrong to use <div> as a root, but React 16 introduced a new feature <Fragment>, import is similar as Component one `import React, { Component, Fragment } from "react";`. As a result the code is a little more performant and also you have a less nested DOM nodes. I think it could be a good idea for future refactoring. You could read more information here: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html